openapi/data_repository_service.swagger.yaml

```diff
@@ -307,7 +292,7 @@ definitions:
           A list of strings that can be used to find this Data Object.
```

Comment:
@susheel -- I like your edits to the Bundle `aliases` field definition. Can you make the same edits to the Object `aliases` field?

Reply:
Done.

Reply:
I don't see any edits? I'll see if I can go ahead and make the change myself to save back and forth.